assets/app/views/images.html

```diff
@@ -10,30 +10,53 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div ng-repeat="imageStream in imageStreams">
-        <div class="tile" ng-repeat="tag in imageStream.status.tags">
+      <div ng-repeat="imageStream in imageStreams track by (imageStream | uid)" class="tile">
+        <h2>
+          {{imageStream.metadata.name}}
+        </h2>
+        <div ng-if="imageStream.spec.dockerImageRepository" class="small">Docker image repository: {{imageStream.spec.dockerImageRepository}}</div>
+        <div ng-if="!(imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck') && !imageStream.status.tags.length"
```

Comment:
Yeah, I think that's better.

![screen_shot_2015-07-17_at_4_21_32_pm](https://cloud.githubusercontent.com/assets/1167259/8756395/ffaf007a-2c9f-11e5-8cc4-0154ed77d217.png)